.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,77 +25,100 @@ IMPORTANT NOTES
 
 <!--The title should clearly define your contribution succinctly.-->
 # Add meaningful title here
-
 <!-- Describe your feature here. Please include any code snippets or examples in this section. -->
 
-## Type of Contribution
+## Section 1: Type of Contribution
 <!-- Check all that apply to help reviewers understand your contribution -->
 - [ ] Feature Enhancement
-  - [ ] New Technology Model
+  - [ ] New Model
```

Comment:
I think maybe either removing the "feature enhancement" or adding another nested option of "other" under "New Model" would make it more clear since right now it seems like new model may be the only possible feature enhancement